### PR TITLE
Add table name to debug on every SQL call

### PR DESCRIPTION
**Pull request by peterbroadhurst**
Currently we take the first three words of the SQL to try and provide a debug statement that's useful, without printing the massive bundle of SQL. However, that is insufficient as for something like `SELECT` it just contains the first two fields from the select.

This PR introduces an explicit table name being passed on every call and put in the logs.

I did find what I think was one bug where `GetContractAPIs` was using the `contract_interfaces` table

### CODE REVIEW

File: internal/database/sqlcommon/contractapis_sql.go [hunk 1]
```diff
@@ -173,7 +175,7 @@ func (s *SQLCommon) GetContractAPIs(ctx context.Context, ns string, filter datab
 		apis = append(apis, api)
 	}
 
-	return apis, s.queryRes(ctx, tx, "contract_interfaces", fop, fi), err
```
Comment (awrichar) [inline internal/database/sqlcommon/contractapis_sql.go:176]: 😳

File: internal/database/sqlcommon/sqlcommon.go [hunk 2]
```diff
@@ -195,7 +177,7 @@ func (s *SQLCommon) queryTx(ctx context.Context, tx *txWrapper, q sq.SelectBuild
 	if err != nil {
 		return nil, tx, i18n.WrapError(ctx, err, coremsgs.MsgDBQueryBuildFailed)
 	}
-	l.Debugf(`SQL-> query: %s`, shortenSQL(sqlQuery))
+	l.Debugf(`SQL-> query %s`, table)
 	l.Tracef(`SQL-> query args: %+v`, args)
```
Comment (awrichar) [inline internal/database/sqlcommon/sqlcommon.go:181]: We logs the args at trace level... but shouldn't we log the full SQL statement as well?

File: internal/database/sqlcommon/sqlcommon.go [hunk 3]
```diff
@@ -226,12 +208,12 @@ func (s *SQLCommon) countQuery(ctx context.Context, tx *txWrapper, tableName str
...
 	}
-	q := sq.Select(fmt.Sprintf("COUNT(%s)", countExpr)).From(tableName).Where(fop)
+	q := sq.Select(fmt.Sprintf("COUNT(%s)", countExpr)).From(table).Where(fop)
 	sqlQuery, args, err := q.PlaceholderFormat(s.features.PlaceholderFormat).ToSql()
 	if err != nil {
 		return count, i18n.WrapError(ctx, err, coremsgs.MsgDBQueryBuildFailed)
 	}
-	l.Debugf(`SQL-> count query: %s`, shortenSQL(sqlQuery))
+	l.Debugf(`SQL-> count query %s`, table)
 	l.Tracef(`SQL-> count query args: %+v`, args)
```
Comment (awrichar) [inline internal/database/sqlcommon/sqlcommon.go:217]: same comment

File: internal/database/sqlcommon/sqlcommon.go [hunk 4]
```diff
@@ -266,25 +248,25 @@ func (s *SQLCommon) queryRes(ctx context.Context, tx *txWrapper, tableName strin
...
 	l := log.L(ctx)
 	q, useQuery := s.provider.ApplyInsertQueryCustomizations(q, requestConflictEmptyResult)
 
 	sqlQuery, args, err := q.PlaceholderFormat(s.features.PlaceholderFormat).ToSql()
 	if err != nil {
 		return i18n.WrapError(ctx, err, coremsgs.MsgDBQueryBuildFailed)
 	}
-	l.Debugf(`SQL-> insert %s`, shortenSQL(sqlQuery))
+	l.Debugf(`SQL-> insert %s`, table)
 	l.Tracef(`SQL-> insert query: %s (args: %+v)`, sqlQuery, args)
```
Comment (awrichar) [inline internal/database/sqlcommon/sqlcommon.go:270]: and here

**Review by awrichar — APPROVED** [review]
Looks great. Just one comment on what's now logged (same comment in multiple places).

File: internal/database/sqlcommon/sqlcommon.go (hunk 2 above)
Comment (peterbroadhurst) [reply internal/database/sqlcommon/sqlcommon.go:181]: Thanks for catching that 👍

**Pull request merged**